apps/old/widget/app/view.jsx

```diff
@@ -81,6 +81,10 @@ const Content = styled.div`
   width: 100%;
   height: 100%;
   background: var(--bg-1, #000);
```

Comment:
Can this background color and .main be moved outside of this component please? 

And video in pull request showing solution please

Reply:
let me try